packages/docusaurus-remark-plugin-npm2yarn/src/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const path = require('path');
+
+module.exports = function (_context, _options) {
+  return {
+    name: 'docusaurus-remark-plugin-npm2yarn',
+    getClientModules() {
+      return [path.resolve(__dirname, './npm2yarn/index.js')];
```

Comment:
I don't think this will work in practice.

We currently don't have any lifecycle method to add a remark plugin to the mdx loader in a generic way.

For now, this package should be a simple remark plugin, not a docusaurus plugin (until we figure this out). We'll use it in the conf by providing it in the remark plugin array.